food-waste-ui/src/components/MarketDetail/MarketDetail.jsx

```diff
@@ -1,3 +1,46 @@
 import * as React from "react";
+import axios from "axios";
```

Comment:
You could split this part into 2 diffs, 1 for using the google map api and one for this market detail